package.json

```diff
@@ -31,25 +31,25 @@
   "author": "steren@google.com",
   "license": "Apache-2.0",
   "devDependencies": {
-    "browser-pack-flat": "^3.4.1",
-    "browserify": "^16.2.3",
-    "chai": "^4.2.0",
-    "core-js": "^2.6.9",
+    "browser-pack-flat": "^3.4.2",
+    "browserify": "^16.5.2",
+    "chai": "^4.3.4",
     "eslint": "^5.16.0",
     "eslint-plugin-mocha": "^5.3.0",
-    "gulp": "^3.9.1",
+    "gulp": "^4.0.2",
     "gulp-rename": "^1.4.0",
-    "gulp-replace": "^1.0.0",
+    "gulp-replace": "^1.1.2",
     "gulp-sourcemaps": "^2.6.5",
     "gulp-uglify": "^3.0.2",
     "http-server": "^0.11.1",
     "mocha": "^5.2.0",
-    "nise": "^1.4.10",
-    "rimraf": "^2.6.3",
+    "nise": "^1.5.3",
+    "rimraf": "^2.7.1",
     "vinyl-buffer": "^1.0.1",
     "vinyl-source-stream": "^2.0.0"
   },
   "dependencies": {
-    "stacktrace-js": "^2.0.0"
+    "core-js": "^3.11.3",
```

Comment:
`core-js` has been moved to devDependencies in https://github.com/GoogleCloudPlatform/stackdriver-errors-js/commit/28f6307799c59b7960d1c7a374b67b173aecb041#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519, can you do the same?

Reply:
@steren ah sorry fixed below